debian/varnish.lintian-overrides

```diff
@@ -11,7 +11,7 @@ varnish: unstripped-binary-or-object usr/bin/varnishstat
 varnish: unstripped-binary-or-object usr/bin/varnishtest
 varnish: unstripped-binary-or-object usr/bin/varnishtop
 
-varnish: unstripped-binary-or-object usr/lib/libvarnishapi.so.2.0.0
+varnish: unstripped-binary-or-object usr/lib/libvarnishapi.so.*
 varnish: unstripped-binary-or-object usr/lib/varnish/vmods/libvmod_*.so
 
 varnish: versioned-provides varnishd-vrt (= 7.0)
```

Comment:
ok, so you've done that for libvarnishapi, can you try with the vrt version? (see my comment on the previous commit)